Osiris/GUI.cpp

```diff
@@ -252,7 +252,7 @@ void GUI::renderLegitbotWindow() noexcept
     ImGui::SetColumnOffset(1, 220.0f);
     ImGui::Checkbox("Aimlock", &config->legitbot[currentWeapon].aimlock);
     ImGui::Checkbox("Silent", &config->legitbot[currentWeapon].silent);
-    ImGui::Checkbox("Friendly fire", &config->legitbot[currentWeapon].friendlyFire);
+    ImGui::Checkbox("Deathmatch mode", &config->legitbot[currentWeapon].friendlyFire);
```

Comment:
FF != deathmatch mode, also you can detect the gamemode and adjust what is ' friendly ' or not based on that